src/homework21/person/PersonSort.java

```diff
@@ -0,0 +1,120 @@
+package homework21.person;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Scanner;
+
+public class PersonSort {
```

Comment:
It should be at least 2 different classes: `Person`, which stores fields to represent data from file and `Main` which contains main logic